kitsune/questions/static/questions/js/components/AAQStep.jsx

```diff
@@ -4,8 +4,11 @@ import scrollTo from '../../../sumo/js/utils/scrollTo.es6.js';
 export default class AAQStep extends React.Component {
   render() {
     let heading = this.heading();
+    let style = {
+      display: this.visible() ? 'block' : 'none'
```

Comment:
Generally instead of using CSS to hide/show content, it works really well to simply return `null`, telling React to put nothing here.